ui/src/test/webapp/app/query/date-range/date-range-component-spec.js

```diff
@@ -1,491 +1,494 @@
-            ctrl.endDate = new Date(-100);
-            ctrl.conf.filter.unit = 'microsecond';
-            ctrl.onEndDateUpdate();
-            expect(dateRange.setEndDate).toHaveBeenCalledWith(-1);
-            expect(dateRange.setEndDate).toHaveBeenCalledTimes(1);
-            expect(serviceEndDate).toEqual(-1);
-        });
-
-        it('should work for Jan 1 1970', function() {
-            ctrl.endDate = new Date(1);
-            ctrl.onEndDateUpdate();
-            expect(dateRange.setEndDate).toHaveBeenCalledWith(86399999);
-            expect(dateRange.setEndDate).toHaveBeenCalledTimes(1);
-            expect(serviceEndDate).toEqual(86399999);
-        });
-
-        it('should work for Jan 1 1970 if units are second', function() {
-            ctrl.endDate = new Date(1);
-            ctrl.conf.filter.unit = 'second';
-            ctrl.onEndDateUpdate();
-            expect(dateRange.setEndDate).toHaveBeenCalledWith(86399);
-            expect(dateRange.setEndDate).toHaveBeenCalledTimes(1);
-            expect(serviceEndDate).toEqual(86399);
-        });
-
-        it('should work for Jan 1 1970 if units are microseconds', function() {
-            ctrl.endDate = new Date(1);
-            ctrl.conf.filter.unit = 'microsecond';
-            ctrl.onEndDateUpdate();
-            expect(dateRange.setEndDate).toHaveBeenCalledWith(86399999999);
-            expect(dateRange.setEndDate).toHaveBeenCalledTimes(1);
-            expect(serviceEndDate).toEqual(86399999999);
-        });
-
-        describe('The end time', function() {
-            beforeEach(function() {
-                ctrl.endDate = new Date(1516620668948);
-            });
-
-            it('should not affect the end date if the time is undefined', function() {
-                ctrl.endTime = undefined;
-                ctrl.onEndDateUpdate();
-                expect(serviceEndDate).toEqual(1516665599999);
-            });
-
-            it('should not affect the end date if the time is null', function() {
-                ctrl.endTime = null;
-                ctrl.onEndDateUpdate();
-                expect(serviceEndDate).toEqual(1516665599999);
-            });
-
-            it('should set the time for the end Date if it exists', function() {
-                ctrl.endTime = new Date(37815000)  // 10:30:15
-                ctrl.onEndDateUpdate();
-                expect(serviceEndDate).toEqual(1516617015000) // date + time
-            });
-
-            it('should round down when using a less precise time unit', function() {
-                ctrl.endTime = new Date(37815000)  // 10:30:15
-                ctrl.conf.filter.unit = 'Day';
-                ctrl.onEndDateUpdate();
-                expect(serviceEndDate).toEqual(Math.floor((1516620668948 + 37815000) / (1000 * 60 * 60 * 24))) // just the date
-            });
-        });
-    });
-});
+// testing of the Date range component will be resumed once time zone is made configurable
```

Comment:
Please can you add a new issue that will address the configurable time zone and then reference the issue number in the code here.